source/mir/random/tinflex/internal/area.d

```diff
@@ -0,0 +1,650 @@
+module mir.random.tinflex.internal.area;
+
+import mir.random.tinflex.internal.types : Interval;
+import std.traits : ReturnType;
+
+/**
+Determines the hat and squeeze function of an interval.
+Based on Theorem 1
```

Comment:
I would always give a minimal citation of the paper, i.e. here `Theorem 1 of Botts et al. (2013)`.  The full citation should be given in module documentation.